lib/cylc/rundb.py

```diff
@@ -709,6 +714,16 @@ def take_checkpoints(self, event, other_daos=None):
                     dao.tables[table_name + "_checkpoints"].add_insert_item(
                         [id_] + list(row))
 
+    def upgrade_from_611(self):
```

Comment:
Shouldn't the upgrade routine be handling the change from `TABLE_TASK_EVENT_HANDLER_TRY_STATES` to `TABLE_TASK_ACTION_TIMERS` as well?